index.bs

```diff
@@ -2729,6 +2769,29 @@ The steps for [=issuing a credential request to an authenticator=] are as follow
 
     1. Return [TRUE].
 
+#### Authentication API Exceptions #### {#sctn-authentication-api-exceptions}
+
+[=[WRPS]=] can encounter the following {{DOMException|DOMExceptions}} from a call to {{CredentialsContainer/get()|navigator.credentials.get()}}:
+
+<dl>
+    :   {{AbortError}}
+    ::  The ceremony was cancelled by an {{AbortController}}.
+        See [[#sctn-abortoperation]] and [[#sctn-sample-aborting]].
+
+    :   {{SecurityError}}
+    ::  The [=effective domain=] was not a [=valid domain=],
+        or <code>{{PublicKeyCredentialCreationOptions/rp}}.{{PublicKeyCredentialRpEntity/id}}</code> was not a registrable domain suffix of nor was equal to the [=effective domain=].
+
+    :   {{UnknownError}}
+    ::  The [=authenticator=] could not process the supplied options,
+        or encountered an error while generating an [=assertion signature=].
+
+    :   {{NotAllowedError}}
+    ::  A catch-all error covering a wide range of possible reasons,
+        including common ones like the user canceling out of the ceremony.
+        Some of these causes are documented throughout this spec,
+        while others are client-specific.
+</dl>
```

Comment:
Is there no TypeError for get credential if the options object is not valid?

it's strange we don't seem to specify that anywhere, but e.g. a wrong type or missing mandatory parameter results in TypeError.

Reply:
Addressing this lack of `TypeError` during auth definitely seems like a fast-follow to this PR.

Reply:
I created https://github.com/w3c/webauthn/issues/2094 to track this.